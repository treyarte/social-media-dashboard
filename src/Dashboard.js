import React, { useState } from 'react';
import SocialMediaCard from './SocialMediaCard';

const Dashboard = () => {
  const [darkMode, setdarkMode] = useState(false);

  const changeMode = () => {
    setdarkMode((darkMode) => !darkMode);
  };

  return (
    <div>
      <SocialMediaCard
        username='@artetech'
        site='twitter'
        icon='tw'
        influence_type='followers'
        influence_count={1231}
        influence_gain={99}
        date='today'
      />
    </div>
  );
};

export default Dashboard;
