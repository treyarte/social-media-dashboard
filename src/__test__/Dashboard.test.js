import React from 'react';
import { render } from '@testing-library/react';
import Dashboard from '../Dashboard';

it('Renders without crashing', () => {
  render(<Dashboard />);
});

it('Matches the snapshot', () => {
  const { asFragment } = render(<Dashboard />);
  expect(asFragment()).toMatchSnapshot();
});
