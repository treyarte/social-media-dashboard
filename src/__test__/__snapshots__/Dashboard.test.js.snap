// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Matches the snapshot 1`] = `
<DocumentFragment>
  <div>
    <div
      class="social-media-card light-card"
    >
      <hr
        class="site-border"
        style="border-top: 10px solid hsl(203, 89%, 53%); border-radius: 20px 20px 0 0;"
      />
      <div>
        <span
          class="tw"
        >
          @artetech
        </span>
      </div>
      <div>
        <h1>
          1231
        </h1>
        <h4>
          followers
        </h4>
      </div>
      <div>
        99 today
      </div>
    </div>
  </div>
</DocumentFragment>
`;
